src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__foto {
+    margin-bottom: 40px;
+    min-height: 134px;
+    padding-inline: 3px;
+  }
+
+  &__title {
```

Comment:
Ensure consistent use of BEM naming conventions. Change `&__title` to `&-title` for consistency.